README.md

```diff
@@ -1,63 +1,98 @@
-# Maslo Companion App
+# Maslo Companion Kit
 
 
 Clone the respository and ensure you have the requirements below.
 
 ### Requirements
 
+ * React Native CLI development environment for both iOS and Android (instructions [`here`](https://reactnative.dev/docs/environment-setup)) 
+ 
  * Node.js 10 ([`nvm`](https://github.com/nvm-sh/nvm) is preferable)
 
  To install:
  ```nvm install 10```
  
  To switch to version 10:
  ```nvm use 10```
+ 
  * Expo CLI
  ```
  npm i -g expo-cli
  ```
+ * React-native CLI
+ ```
+ npm i -g react-native-cli
+ ```
  * Firebase Tools:
  ```
  npm i -g firebase-tools
  ```
-### Clone external dependencies
-
-1. Create a folder inside of the ```mobile``` folder called ```dependencies``` and navigate inside it.
-2. Clone both [Maslo Persona](https://github.com/HeyMaslo/maslo-persona/tree/dev-ts) from the dev-ts branch and [React Native Switch Pro](https://github.com/HeyMaslo/react-native-switch-pro) from the master branch inside the ```mobile/dependencies``` folder.
-3. Now rename the ```maslo-persona``` folder that you cloned in the previous step to ```persona```.
+ 
 
 
-### Install dependencies and validate Node.js version
+## Configuration Setup
+
+1. Copy the values from the `firebaseConfig` variable in the web app in the firebase console to the `FirebaseConfigs` variable in `./config/app.js`.
+2. Download the `GoogleService-Info.plist` file from the ios app on firebase. Place this file in `./mobile/configs/app`.
+3. Copy the `GOOGLE_APP_ID` and `API_KEY` from the plist file you just downloaded into the ios object in the `FirebasePlatformOverrides` variable in `./config/app.js`. Then copy the `CLIENT_ID` to line 48 in the `GoogleConfigs` variable in the same file.
+4. Copy the `REVERSED_CLIENT_ID` from the same plist file into line 39 in `./mobile/app.json`.
```

Comment:
@cara-wong Weird question and not something we should change but do you know why one is called **reversed** and the other is called **reserved**?

Reply:
Not a weird question - I was wondering that too. I'm not sure why that is